migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql

```diff
@@ -0,0 +1,50 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- IDEMPIERE-2955 Adding Attribute set type field (M_AttributeSet_Type), support DMS/MM
+ Dec 2, 2015 11:53:13 AM IST
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,PrintName,EntityType,AD_Element_UU) VALUES (202952,0,0,'Y',TO_TIMESTAMP('2015-12-02 11:53:12','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:53:12','YYYY-MM-DD HH24:MI:SS'),100,'M_AttributeSet_Type','AttributeSet Type','AttributeSet Type','D','7b193413-66ff-4a9b-a4e3-73cacc8d413d')
+;
+
+-- Dec 2, 2015 11:53:49 AM IST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,DefaultValue) VALUES (212554,0,'AttributeSet Type',560,'M_AttributeSet_Type',3,'N','N','Y','N','N',0,'N',17,0,0,'Y',TO_TIMESTAMP('2015-12-02 11:53:44','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:53:44','YYYY-MM-DD HH24:MI:SS'),100,202952,'Y','N','D','N','N','N','Y','b0f70177-747f-4e25-9d88-df4111f43712','Y',0,'N','N','MMS')
+;
+
+-- Dec 2, 2015 11:54:23 AM IST
+INSERT INTO AD_Reference (AD_Reference_ID,Name,ValidationType,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,IsOrderByValue,AD_Reference_UU) VALUES (200115,'AttributeSet Type','L',0,0,'Y',TO_TIMESTAMP('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,'D','N','dc52c0cb-adec-4c9e-a176-29ca09f0072e')
+;
+
+-- Dec 2, 2015 11:56:12 AM IST
+INSERT INTO AD_Ref_List (AD_Ref_List_ID,Name,AD_Reference_ID,Value,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,AD_Ref_List_UU) VALUES (200344,'Material Management System',200115,'MMS',0,0,'Y',TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,'D','4455f892-2ea1-499d-a0b9-c01b3226f97d')
+;
+
+-- Dec 2, 2015 11:56:29 AM IST
+UPDATE AD_Column SET AD_Reference_Value_ID=200115,Updated=TO_TIMESTAMP('2015-12-02 11:56:29','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212554
```

Comment:
There are 9 TO_TIMESTAMP here and above - the correct syntax in oracle is TO_DATE